.pipelines/azure_pipeline_prod.yaml

```diff
@@ -182,28 +183,99 @@ jobs:
         az account set -s ${{ variables.subscription }}
         az acr login -n ${{ variables.containerRegistry }}
 
-        @{"image.name"="${{ variables.repoImageNameWindows }}:$(windowsImageTag)"} | ConvertTo-Json -Compress | Out-File -Encoding ascii $(Build.ArtifactStagingDirectory)/windows/metadata.json
+        docker build --isolation=hyperv --tag ${{ variables.repoImageName }}:$(windowsImageTag)-$(windows2019BaseImageVersion) --build-arg WINDOWS_VERSION=$(windows2019BaseImageVersion) --build-arg IMAGE_TAG=$(windowsImageTag) .
+        if ("$(Build.Reason)" -ne "PullRequest") {
+           docker push ${{ variables.repoImageName }}:$(windowsImageTag)-$(windows2019BaseImageVersion)
+        }
+
+- job: build_windows_2022
+  dependsOn:
+  -  common
+  pool:
+    name: Azure-Pipelines-Windows-CI-Prod-EO
```

Comment:
for the prod pipeline agent pool I don't think we have added the 2022 image

Reply:
> for the prod pipeline agent pool I don't think we have added the 2022 image

> Yes, I haven't added yet. Looks like I don't have permission sub where are these resources exists.lets sync how to get the access.

Reply:
> > for the prod pipeline agent pool I don't think we have added the 2022 image
> 
> > Yes, I haven't added yet. I don't have permission to sub where are these resources exists. Lets sync how to get the access.

> created ws2022 compliant image and updated the pipeline to use this image.

Reply:
> > > for the prod pipeline agent pool I don't think we have added the 2022 image
> > 
> > 
> > > Yes, I haven't added yet. I don't have permission to sub where are these resources exists. Lets sync how to get the access.
> 
> > created ws2022 compliant image and updated the pipeline to use this image.

>> validate prod pipeline works as expected.